pkgs/development/python-modules/pytest/default.nix

```diff
@@ -75,13 +75,12 @@ buildPythonPackage rec {
     preDistPhases+=" pytestcachePhase"
   '';
 
-  pythonImportsCheck = [
-    "pytest"
-  ];
+  pythonImportsCheck = [ "pytest" ];
```

Comment:
Please undo this change.
Having items on a separate line is preferable as it makes merge conflicts easier to resolve.

Reply:
Expanding the list of import checks of doesn't make much sense as they will all fail if `pytest` can't be imported.

The "one item per line" doesn't correspond with other parts of the file (e.g. the `checkInputs` or the imports).

Reverted and implemented suggestions from `nixpkgs-hammering`.